site/multiDim/MultiDimDataPage.tsx

```diff
@@ -31,19 +31,21 @@ export function MultiDimDataPage({
         relatedResearchCandidates,
         imageMetadata,
         tagToSlugMap,
-        initialQueryStr,
     }
+    const imageUrl: string = urljoin(baseUrl, "default-grapher-thumbnail.png")
```

Comment:
This isn't getting overwritten correctly in staging it seems

<img width="809" alt="image" src="https://github.com/user-attachments/assets/148e0848-9eae-4015-a174-9f4687d3d331" />

I can't debug it correctly because my local version of the MDD previews don't render when served through the wrangler. Also, even when served by the admin server I get `"No table loaded yet"` - do you know what's going on there?

Reply:
I have some ideas on what might be going on with your local dev setup, and it would be easiest if we could have a call to debug it together. Short summary:

* Make sure you have the latest code and data in the DB
* To see the normal mdim page at `/grapher` (not the admin preview) you need to set `published` to `1` in `multi_dim_data_pages` and run the migration script, so it propagates to chart configs with:

  ```
  yarn tsx --tsconfig tsconfig.tsx.json devTools/migrateMultiDimConfigs/migrateMultiDimConfigs.ts
  ```
  
  (I'll be adding an admin for publishing mdims later this cycle.)